ads/google/doubleclick.js

```diff
@@ -55,39 +55,20 @@ export function doubleclick(global, data) {
     };
   }
 
-  // Center the ad in the container.
-  const container = global.document.querySelector('#c');
-  setStyles(dev().assertElement(container), {
-    top: '50%',
-    left: '50%',
-    bottom: '',
-    right: '',
-    transform: 'translate(-50%, -50%)',
-  });
+  centerAd();
+
+  const url = getUrl(data);
+
+  whichGladeExperimentBranch(data, url, experimentFraction);
 
-  if (data.useSameDomainRenderingUntilDeprecated != undefined ||
-      data.multiSize) {
-    doubleClickWithGpt(global, data, GladeExperiment.GLADE_OPT_OUT);
-  } else {
-    const dice = Math.random();
-    const href = global.context.location.href;
-    if ((href.indexOf('google_glade=0') > 0 || dice < experimentFraction)
-        && href.indexOf('google_glade=1') < 0) {
-      doubleClickWithGpt(global, data, GladeExperiment.GLADE_CONTROL);
-    } else {
-      const exp = (dice < 2 * experimentFraction) ?
-        GladeExperiment.GLADE_EXPERIMENT : GladeExperiment.NO_EXPERIMENT;
-      doubleClickWithGlade(global, data, exp);
-    }
-  }
 }
 
 /**
- * @param {!Window} global
  * @param {!Object} data
  * @param {!GladeExperiment} gladeExperiment
+ * @param {!string} url
  */
-function doubleClickWithGpt(global, data, gladeExperiment) {
+function doubleClickWithGpt(data, gladeExperiment, url) {
```

Comment:
I changed the signature of that function as a way to make the testing easier. But since I've now punted on implementing those tests for now I changed it back to its original form.